### ISSUE TITLE
PR might remove window.Chessground

### ISSUE THREAD

**Issue by Siderite** [Siderite/lichessTools#167]
Make sure the mobile drawing of arrows still works after next deploy.

**Comment by Siderite** [comment]
Use code like
```
const makeCg = (await import('https://lichess1.org/blah/blah/chessground.min.js')).Chessground;
const myApi = makeCg(myEl, myConfig);
```
to get an instance of Chessground

**Comment by Siderite** [comment]
fixed in V2.1.2

**Comment by Siderite** [comment]
reopened because a new mechanism might be created using a makeChessground method.

**Comment by Siderite** [comment]
fixed in v2.1.4
